src/go/rdctl/cmd/set.go

```diff
@@ -55,6 +60,7 @@ func init() {
 	setCmd.Flags().StringVar(&specifiedSettings.ContainerEngine, "container-engine", "", "Set engine to containerd or moby (aka docker).")
 	setCmd.Flags().BoolVar(&specifiedSettings.Enabled, "kubernetes-enabled", false, "Control whether kubernetes runs in the backend.")
 	setCmd.Flags().StringVar(&specifiedSettings.Version, "kubernetes-version", "", "Choose which version of kubernetes to run.")
+	setCmd.Flags().BoolVar(&specifiedSettings.Flannel, "flannel-enabled", true, "Control whether kubernetes runs in the backend.")
```

Comment:
Looks like the explanation is a copy/pasto of `--kubernetes-enabled`. Would be great if you could sum up the reason/consequence of disabling this for an average user.